ripe/atlas/tools/helpers/validators.py

```diff
@@ -47,25 +37,90 @@ def datetime(string):
                 "UTC."
             )
 
+    @staticmethod
+    def ip_or_domain(string):
+        message = '"{}" does not appear to be an IP address or host ' \
+                  'name'.format(string)
+
+        if " " in string:
+            raise argparse.ArgumentTypeError(message)
+        if "." not in string and ":" not in string:
+            if not re.match(r"^\w+$", string):
+                raise argparse.ArgumentTypeError(message)
+
+        return string
+
```

Comment:
I think we shouldn't go that far and do these kind of validations. Let users fail with their mistakes. It's an never ending road if we start doing validations, and we duplicate code since we have validations on API itself:) But since you have that now I merge it, but let's not over do it :) What say you?
I like that there is usually a tests section lately :)